Docs/order_gfa.md

```diff
@@ -0,0 +1,26 @@
+# Ordering Graph
+
+This subcommand establishes an order to the graph based on the "bubbles" in the graph. 
+Here, we define the bubbles as biconnected components, i.e. not the strict definition of a bubble found in other papers.
+
+The graph input here has to be an [rGFA](https://github.com/lh3/gfatools/blob/master/doc/rGFA.md), with SN and SO tags.
+
+The basic idea here is that we first detect all biconnected components (bubbles), and collapse the detected bubbles into one node,
+which should result in a line graph made from scaffold nodes that belong to the reference and the collapsed bubbles inbetween. We then
+order this line graph in ascending order based on the coordinates in the SO tag. Each node in this ordered line graph 
+gets an ascending BO tag from 1 to N (N being the number of nodes in the line graph). For the nodes that represent a collapsed
+bubbles, all the nodes in that bubble will get the same BO tag (Figure 1). As for the NO tag, the nodes in a bubble get an ascending 
+number from 1 to M (M being the number of nodes in a bubble). However, the NO tag inside a bubble is assigned based on the node id order, i.e.
+in somewhat random order.
```

Comment:
Is this not a node-id lexicographic order?

Reply:
Yes, inside a bubble, the NO tags are given in lexicographic order based on the node id

Reply:
Ok. Then I would suggest to specify that so that NO tags generated by someone else's scripts and our script will match.